proposals/0569-multiline-strings.rst

```diff
@@ -117,7 +117,7 @@ A working prototype is available at `brandonchinn178/string-syntax <https://gith
 
    #. Join the string back with ``\n`` delimiters
 
-      * Use ``\n`` regardless of the line terminators being used in the file. This matches the behavior of ``unlines``.
+      * Use ``\n`` regardless of the lexical ``newline``s present in the file. This matches the behavior of ``unlines``.
```

Comment:
it looks like the `s` immediately following the inline code messes up the formatting. This seems to fix it.
```suggestion
      * Use ``\n`` regardless of the lexical ``newline``\ s present in the file. This matches the behavior of ``unlines``.
```

Reply:
does this work:
```
``newline``(s)
```

Reply:
I tried it in a gist, but it looks like that is also formatted incorrectly